legacy_resources/README.md

```diff
@@ -0,0 +1,474 @@
+# Ember-sortable
+
+[![npm version](https://badge.fury.io/js/ember-sortable.svg)](http://badge.fury.io/js/ember-sortable)
+[![Build Status](https://travis-ci.org/adopted-ember-addons/ember-sortable.svg?branch=master)](https://travis-ci.org/adopted-ember-addons/ember-sortable)
+[![Downloads Weekly](https://img.shields.io/npm/dw/ember-sortable)](http://badge.fury.io/js/ember-sortable)
+[![Ember Observer Score](http://emberobserver.com/badges/ember-sortable.svg)](http://emberobserver.com/addons/ember-sortable)
+[![Code Climate](https://codeclimate.com/github/jgwhite/ember-sortable/badges/gpa.svg)](https://codeclimate.com/github/jgwhite/ember-sortable)
+
+Sortable UI primitives for Ember.
+
+![ember-sortable in action](/demo.gif)
+
+[Check out the demo](https://adopted-ember-addons.github.io/ember-sortable/demo/)
+
+## Migration
+If you are migrating from `1.x.x` to `2.x.x`. Please read this [migration guide](/MIGRATION_GUIDE_V2.md).
+
+## Requirements
+
+In version 2.0.0+, our `closest` polyfill seems to break some app's `production` build. To mitigate this, the `closest` polyfill will only enabled if it doesn't break the `production` build (if the `polyfill` file is recognized by the build). Affected apps will need to supply their own [closest](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#Polyfill) polyfill to ensure compatibility with IE. This issue is tracked [here](https://github.com/adopted-ember-addons/ember-sortable/issues/333).
+
+Version 1.0 depends upon the availability of 2D CSS transforms.
+Check [the matrix on caniuse.com](http://caniuse.com/#feat=transforms2d)
+to see if your target browsers are compatible.
+
+## Installation
+
+```sh
+$ ember install ember-sortable
+```
+
+## Usage
+
+### component 
+```hbs
+{{! app/templates/my-route.hbs }}
+
+{{#sortable-group model=model.items onChange=(action "reorderItems") as |group|}}
+  {{#each group.model as |modelItem|}}
+    {{#group.item model=modelItem as |item|}}
+      {{modelItem.name}}
+      {{#item.handle}}
+        <span class="handle">&varr;</span>
+      {{/item.handle}}
+    {{/group.item}}
+  {{/each}}
+{{/sortable-group}}
+```
+
+### modifier 
+```hbs
+{{! app/templates/my-route.hbs }}
+
+<ol {{sortable-group onChange=(action "reorderItems")}}>
+  {{#each model.items as |modelItem|}}
+    <li {{sortable-item model=modelItem}}>
+      {{modelItem.name}}
+      <span class="handle" {{sortable-handle}}>&varr;</span>
+    </li>
+  {{/each}}
+</ol>
+```
```

Comment:
If the modifier version is going to be released as 3.x.x, probably we probably should remove references to in the V2 migration guide since I dont think it was ever released in a 2.x series